modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -62,6 +81,44 @@ components:
                 -   actionIds
                 -   type
             type: object
+        ObjectEntryCMSBulkActionRequest:
+            description:
+                Represents the request for CMS bulk actions
+            properties:
+                ids:
+                    items:
+                        format: int64
+                        type: integer
+                    type: array
+                keywords:
+                    description:
+                        A list of keywords describing the account.
+                    items:
+                        type: string
+                    type: array
+                objectEntryFolderId:
```

Comment:
We think the Request schema should not contain all these data, independently of the action that is going to be executed. For example, for the remove action, the `objectEntryFolderId` is not revelant at all. Furthermore, for any user, it is really difficult to understand the request schema that must be sent just by checking the OpenAPI.

Our purpose is to create as many endpoints as actions exist. For example:

`/object-entry-cms-bulk-action/remove` with a request body containing only the data relevant for the remove
`/object-entry-cms-bulk-action/move` with a request body containing only the data relevant for the remove

etc